recipe/meta.yaml

```diff
@@ -1,62 +1,71 @@
-{% set version = "2.3.8" %}
+{% set version = "5.6.0" %}
 
 package:
   name: pymc
   version: {{ version }}
 
 source:
   url: https://github.com/pymc-devs/pymc/archive/v{{version}}.tar.gz
-  sha256: 200a28469d574a370fa9b55cd92f112f8ad09f71674bdae514437039e953689b
+  sha256: a09b11a0ae9a56a6f57b9b89f161d06b1d49b7d80a84311eba4376f2a32b5d0f
 
 build:
   number: 0
+  # skip because arviz is not supported for py38
+  skip: True  # [py<39]
   script:
     - export LDFLAGS="-undefined dynamic_lookup -bundle"  # [osx]
     - export LDFLAGS="$LDFLAGS -shared"  # [linux]
```

Comment:
You could use script_env attribute to define env variables, like here:
https://github.com/AnacondaRecipes/flit-feedstock/blob/master/recipe/meta.yaml#L47

Reply:
thanks!